packages/llvm.rb

```diff
@@ -21,7 +21,6 @@ class Llvm < Package
   })
 
   depends_on 'python27' => :build # for test suite
-  depends_on 'graphviz' => :build
```

Comment:
You sure this is not needed?  This is required for the `dot` command.  It's only a build dependency also.

Reply:
Yes, but it can be disabled and it causes a load of dependency loops.

Reply:
It shouldn't cause a dependency loop as a build dependency or are you saying it causes the looping when you try to compile?

Reply:
When I try to compile.